manifests/cli.pp

```diff
@@ -0,0 +1,113 @@
+# @summary Manage Sensu CLI
+#
+# Class to manage the Sensu CLI.
+#
+# @example
+#   class { 'sensu::cli':
+#     password => 'secret',
+#   }
+#
+# @param version
+#   Version of sensu-go-cli to install.  Defaults to `installed` to support
+#   Windows MSI packaging and to avoid surprising upgrades.
+# @param package_name
+#   Name of Sensu CLI package.
+# @param install_source
+#   Source of Sensu Go CLI download for installing on Windows.
+#   Paths with http:// or https:// will be downloaded
+#   Paths with puppet:// or file:// paths will also be installed.
+# @param install_path
+#   Where to install sensuctl for Windows. Default to `C:\Program Files\Sensu`.
+# @param url_host
+#   Sensu backend host used to configure sensuctl and verify API access.
+# @param url_port
+#   Sensu backend port used to configure sensuctl and verify API access.
+# @param password
+#   Sensu backend admin password used to confiure sensuctl.
+# @param bootstrap
+#   Should sensuctl be bootstrapped. This is a private parameter used by sensu::backend class.
```

Comment:
it is used in this class, too

Reply:
I found a way to avoid bootstrap parameters. This has been removed.